developers/wallets/index.md

```diff
@@ -68,11 +76,8 @@ You can register a Credential Handler by calling the `CredentialManager.requestP
   target="_blank" rel="noopener noreferrer"> chapi-demo-wallet/install-wallet.js </a>
 </p>
 
-### Then, configure your wallet to respond to Credential Handler events.
-Your wallet app's existing functionality can be configured to respond to CHAPI events.
-
-#### 4. Setup Listeners for CHAPI Events
-The `activateHandler()` function is a helper that sets up listeners for CHAPI `get()` and `store()` events.
+### 4. Setup Listeners for CHAPI Events
+Configure your wallet to respond to Credential Handler events. The `activateHandler()` function is a helper that sets up listeners for CHAPI `get()` and `store()` events.
```

Comment:
We should be clear that there are two ways to receive requests from CHAPI, one is via CHAPI events and the other via URLs (which can be deep / "app" links).